linux/sockstat.go

```diff
@@ -0,0 +1,94 @@
+package linux
+
+import (
+	"io/ioutil"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+type Sockstat struct {
```

Comment:
> Done in f1caad9.

Hmm. I remember why I didn't camel case this in the first place. I was trying to keep it close to the actual name in proc: /proc/net/sockstat. Just like `Netstat`.